User-Help/FormattingHelp.md

```diff
@@ -0,0 +1,107 @@
+
+* **Monospace / code formatting**
+
+  To make a word monospace, you can surround it with a single backtick (<code>&#96;</code>). This is generally used for code formatting. (Some people use this for emphasis. Don't do that! Screen readers will read out words formatted as code character by character - use *italics* or **bold** for emphasis instead.)
+
+  To mark several lines as code, you can use a code fence. This is three backticks on a newline both on top and at the bottom of your lines of code. Don't use this for quotes - see below for guidance on quote formatting.
+
+* **Links**
+
+  You can generate a link using one of two methods. One way is to encase your link text in square brackets (`[ ]`) and then the target URL in parentheses (`( )`). `[Codidact](https://codidact.org)` will render as [Codidact](https://codidact.org).
+
+  You can also format a link with the inline text being `[Codidact][1]`, and then at the bottom of the post include the URL with the format `[1]: https://codidact.org`. This will look something like this:
+
+```
+
+  You can check out [Codidact][1] for an open-source, community-oriented Q&A platform!
+  
+  [1]: https://codidact.org`
+
+```
+  Which will render like this:
+
+  > You can check out [Codidact][1] for an open-source, community-oriented Q&A platform!
+  > 
+  > [1]: https://codidact.org
+
+  If you'd like to include rollover text for your link, you can put your text, surrounded by quote marks (`"`), into the URL of your link. For instance, `[Codidact](https://codidact.org "The open-source Q&A platform")` will create a link with `The open-source Q&A platform` as the rollover text.
+
+* **Blockquotes**
+
+  When quoting text from somewhere else, you can use a blockquote to signify that this is not your original text, but a quote from somewhere else. Put a greater than symbol (`>`), followed be a space, before a paragraph to mark that quote as a blockquote. For instance, `> This is a blockquote.` will render as a quote:
+
+  > This is a blockquote!
+
+  If you want to include a second paragraph as part of the same quote, you have to include another `>` before each line, like this:
+
+```
+  > This is a blockquote!
+  >
+  >This is part of the same quote!
+```
+
+  This will render as a single blockquote:
+
+  > This is a blockquote!
+  >
+  > This is part of the same quote!
+
+  See also the [citation and referencing help](/help/referencing) for guidelines on how to properly reference different quotes from various places.
+
+* **Images**
+
+  Embedding images in a post is very similar to the formatting for links (detailed above). The format `![alt text](image URL)` will embed an image in your post. Please remember to write alt text for your image! Codidact allows you to upload an image directly, or you can embed an external image from image hosting sites such as Imgur.
```

Comment:
Do we have any documentation on image restrictions yet (file size, rendered dimensions, formats)?  If so, this would be a good place to link it.

Reply:
No documentation, but the image uploader has a comment saying "max size 2MB".

Reply:
If the UI communicates it already then I don't think we need to put it here too.